src/lib/wallet/utils.js

```diff
@@ -46,6 +47,13 @@ export const toRawValue = (masked: string, settings?: {}): number =>
 export const weiToMask = (wei: number, settings?: {}): string => toMask(weiToGd(wei), settings)
 export const maskToWei = (mask: string, settings?: {}): number => gdToWei(toRawValue(mask, settings))
 
+/**
+ * generates function that formats numeric values to its abbreviation ie 1000 = 1k
+ * @param {number} decPlaces
+ * @returns {(wei: number) => string}
+ */
+export const abbreviateValue = decPlaces => wei => abbreviate(weiToGd(wei), decPlaces)
```

Comment:
1. we already have helpers to abbreviate number. formatWithSIPrefix or something like this
2. see if you could use our helpers
3. update our helpers to use number-abbreviate package